tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BCompileUtil.java

```diff
@@ -70,6 +72,17 @@ public static CompileResult compile(String sourceFilePath) {
         return compileResult;
     }
 
+    public static BPackageSymbol generateBIR(String sourceFilePath) {
```

Comment:
Is it the symbol we need to return or the BIRPackageFile?

Reply:
I think best would be to move it [BIRCompileResult](https://github.com/ballerina-platform/ballerina-lang/blob/62191c2566e9e35121d1169e6fc57021ef0b962a/docs/bir-spec/src/test/java/org/ballerinalang/birspec/BIRTestUtils.java#L646) into BCompileUtil and return that. WDYT?

Reply:
+1